spec/models/user_spec.rb

```diff
@@ -63,4 +63,9 @@
     it { should validate_length_of(:first_name).is_at_most(250) }
     it { should validate_length_of(:last_name).is_at_most(250) }
   end
+
+  context 'user admin' do
```

Comment:
why context?
this should be a desribe "factory", context :admin trait or something